app/[contentType]/[slug]/page.tsx

```diff
@@ -0,0 +1,189 @@
+import { Metadata } from 'next';
+import Image from 'next/image';
+import { notFound } from 'next/navigation';
+import { Chips, Container } from '../../../components';
+import BackButton from '../../../components/back-button';
+import { author, site } from '../../../config/index.json';
+import {
+  getContentData,
+  getContentList,
+  getContentTypes,
+  IContentType,
+} from '../../../utils/content';
+import { contentTypesMap } from '../../../utils/content-types';
+import Content from './content';
+
+export async function generateMetadata({
+  params,
+}: {
+  params: Params;
+}): Promise<Metadata> {
+  const { title, previewImage, description } = await getContentData(
+    params.slug,
+    params.contentType,
+  );
+  return {
+    title: `${title} | ${site.siteTitle}`,
+    description: description ?? site.siteDescription,
+    openGraph: {
+      title: `${title} | ${site.siteName}`,
+      description: description ?? site.siteDescription,
+      url: site.siteUrl,
+      images: previewImage ?? site.siteImage,
+      siteName: site.siteName,
+    },
+    twitter: {
+      card: 'summary_large_image',
+      creator: author.twitterHandle,
+      images: previewImage ?? site.siteImage,
+    },
+  };
+}
+
+/**
+ * statically generate all content pages
+ */
+export function generateStaticParams() {
+  const contentTypes = getContentTypes();
+
+  return contentTypes.flatMap((contentType) => {
+    const contentList = getContentList(contentType);
+    return contentList.map(({ slug }) => {
+      return {
+        contentType,
+        slug,
+      };
+    });
+  });
+}
```

Comment:
Changed `contentTypes.map().flat()` to `contentTypes.flatMap()`

Reply:
I'm assuming this was also from a linting warning, right?

Reply:
Yes it was